discovery.js

```
@@ -284,7 +290,7 @@ DiscoveryClient.prototype.announce = function (announcement, cb) {
 
 /* Remove a previous announcement.  The passed object *must* be the
  * lease as returned by the 'announce' callback. */
-DiscoveryClient.prototype.unannounce = function (announcement, callback) {
+DiscoveryClient.prototype.unannounce = function (announcement, cb) {
   var disco = this;
   var server = disco._randomServer();
```

I agree but instead of checking if server is defined we can just throw an error inside the method _randomServer